core/src/test/java/com/linecorp/armeria/client/logging/LoggingClientTest.java

```diff
@@ -95,4 +99,59 @@ void logger() throws Exception {
         defaultLoggerClient.execute(ctx, req);
         verifyNoInteractions(logger);
     }
+
+    @Test
+    void sanitizeRequestHeaders() throws Exception {
+        final HttpRequest req = HttpRequest.of(RequestHeaders.of(HttpMethod.POST, "/hello/trustin",
+                                                                 HttpHeaderNames.SCHEME, "http",
+                                                                 HttpHeaderNames.AUTHORITY, "test.com"));
+
+        final ClientRequestContext ctx = ClientRequestContext.of(req);
+
+        // use default logger
+        final LoggingClient defaultLoggerClient =
+                LoggingClient.builder()
+                             .requestLogLevel(LogLevel.INFO)
+                             .successfulResponseLogLevel(LogLevel.INFO)
+                             .requestHeadersSanitizer(new RegexBasedSanitizerBuilder()
+                                                              .pattern("trustin")
+                                                              .pattern("com")
+                                                              .build())
+                             .build(delegate);
+
+        // Pre sanitize step
+        Assertions.assertTrue(ctx.logBuilder().toString().contains("trustin"));
+        Assertions.assertTrue(ctx.logBuilder().toString().contains("test.com"));
```

Comment:
Sorry, but we use AssertJ for assertions. Could you convert to AssertJ? e.g.

    assertThat(ctx.logBuilder().toString()).contains("trustin");
    ...
    assertThat(ctx.logBuilder().toString()).doesNotContain("trustin");

Reply:
Done